proto/process/agent.proto

```diff
@@ -110,6 +110,93 @@ message CollectorReqStatus {
 	string hostName = 2;
 }
 
+message CollectorECSTask {
+	int64 awsAccountID = 1;
+	string clusterName = 2;
+	string clusterId = 3;
+	string region = 4;
+	int32 groupId = 5;
+	int32 groupSize = 6;
+	repeated ECSTask tasks = 7;
+}
+
+message ECSTask {
+	string arn = 1;
+	string taskVersion = 2;
+	string launchType = 3;
+	string desiredStatus = 4;
+	string knownStatus = 5;
+	string family = 6;
+	string version = 7;
+	string availabilityZone = 8;
+	map<string, double> limits = 9;
```

Comment:
Yes. I feel I still prefer to use a map instead of using two dedicated fields as we can check if the value is set or not
```
if val, ok := limits["CPU"]; ok {
			//do somthing
}
```

Reply:
maybe we should define some constants for the keys?

Reply:
we can't use enum type as key for a map in proto 😞 
It's a usage we are already using https://github.com/DataDog/agent-payload/blob/master/proto/process/agent.proto#L996

Reply:
It seems it's impossible to define constant strings in protobuf3, and it's not possible to check if a field is has a value or not since numeric types are initiated by 0 in golang.
So leaving this as is is the best option I think